great_expectations/validator/resolved_metric.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Dict, List, Set, Tuple, Union
+
+import numpy as np
+import pandas as pd
+
+MetricValue = Union[
+    Any,  # Encompasses deferred-query/execution plans ("SQLAlchemy" and "Spark") conditions and aggregation functions.
```

Comment:
Is it in fact a "resolved metric" if it can be a SQLAlchemy or Spark object? I thought we would call this something like `UnresolvedMetric`.

Reply:
@NathanFarmer You are right.  It can be fully resolved or partially resolved.  Let me come up with a better name.